russian/programming/to-program.md

```diff
@@ -1,5 +1,9 @@
+
 Вы решаете, что в любой непонятной ситуации нужно писать код, и отправляетесь на поиски комьютера.
 
 [Искать в другой комнате](../next_room/next_room.md)
 
-[Поработать на компьютере соседей](../speak_to_neighbor/speak_to_neighbor.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.